src/library.js

```diff
@@ -3270,11 +3270,11 @@ LibraryManager.library = {
 #if ASSERTIONS
       assert(ch === 100/*'d'*/ || ch === 102/*'f'*/ || ch === 105 /*'i'*/);
 #endif
-      // A double takes two 32-bit slots, and must also be aligned - the backend
+      // A double or readAsmConstArgsDouble takes two 32-bit slots, and must also be aligned - the backend
```

Comment:
Did you mean o change the comment here?

Reply:
I just put the `or` there in case someone became confused as to what `readAsmConstArgsDouble` was.

Reply:
I don't think you want the "or readAsmConstArgsDouble" here since "A double" is referring the a type, not a variable name.

Reply:
Ok I can revert it

Reply:
Will hold off until we have this resolved